manifests/init.pp

```diff
@@ -131,11 +132,88 @@
   Boolean                       $use_internet_repo       = false,
   String                        $version                 = simplib::lookup('simp_options::package_ensure', { 'default_value' => 'installed' }),
   String                        $rpm_iteration           = '1',
-  Boolean                       $simp_dashboards         = false
-) inherits ::simp_grafana::params {
+  Boolean                       $simp_dashboards         = false,
+  String                        $admin_pw                = passgen('grafana'),
+  String                        $app_pki_dir             = '/etc/pki/simp_apps/grafana/x509',
+  String                        $app_pki_key             = "${app_pki_dir}/private/${facts['fqdn']}.pem",
+  String                        $app_pki_cert            = "${app_pki_dir}/public/${facts['fqdn']}.pub",
+ ) inherits ::simp_grafana::params {
 
-  $merged_cfg = deep_merge($::simp_grafana::params::cfg, $cfg)
-  $merged_ldap_cfg = deep_merge($::simp_grafana::params::ldap_cfg, $ldap_cfg)
+  # Static defaults
+  $_cfg = {
```

Comment:
I'm fairly certain that this can be moved to data in modules as long as you make the `$cfg` parameter last in the list above. @heliocentric may have more specific information on the viability of this.